apis/kueue/v1beta1/workload_types.go

```diff
@@ -580,6 +580,10 @@ const (
 	// because the LocalQueue is Stopped.
 	WorkloadEvictedByLocalQueueStopped = "LocalQueueStopped"
 
+	// WorkloadEvictedDueToNodeFailures indicates that the workload was evicted
+	// due to non-recoverable node failures.
+	WorkloadEvictedDueToNodeFailures = "WorkloadEvictedDueToNodeFailures"
```

Comment:
Why not just use `NodeFaulures`? IIUC, isn't this for the Eviction condition type reason?

Reply:
Yes, the condition type already sets the context

Reply:
A built condition will be the following. The condition has `Evict` term in 2 places. It looks like too redundant.

```yaml
status:
  conditions:
  - type: Evicted
    reason: WorkloadEvictedDueToNodeFailures
    status: True
    message: xyz
```

Reply:
SGTM, thanks